### DISCUSSION TITLE
LicenseManager

### DISCUSSION THREAD

**Discussion by sMOTIVEEd** [primefaces/community#836] [PrimeNG]
Using the LicenseManager on version **16.9.3-lts** with any other host than localhost results in the **not licensed** banner.

The console shows the following:
`Error: Cannot read properties of undefined (reading 'digest') primeng-api.mjs:979`

At the moment we can't ship any version to our customers. Is there a work-around?

**Comment by cetincakiroglu** [comment]
Hi,

Could you please use the lines below and get back to us with the error code?

```
    LicenseManager.verify('LICENSE_KEY', 'PASS_KEY', true);
    // true enables the debugging and makes easier to detect broken part of the licenseManager
```

**Reply by sMOTIVEEd** [reply]
Hello,
the console-log mentioned in my initial message was already generated with the debug-flag set to **true**.
Sorry, there no other messages.

**Reply by cetincakiroglu** [reply]
Hi,

Could you please contact and send your credentials by email to contact@primetek.com.tr
You can also use the PRO support platform and create an issue ticket.
